addons/api/package.json

```diff
@@ -28,11 +28,11 @@
     "ember-cli-htmlbars": "^6.2.0",
     "ember-cli-mirage": "^2.4.0",
     "ember-data": "~4.12.0",
+    "@babel/core": "^7.21.0",
```

Comment:
Why was this added to dependencies from dev?

Reply:
@ZedLi fixed